### PR TITLE
Added buttons

### PR DESCRIPTION
**Pull request by matthewleemle**

### CODE REVIEW

File: screens/ResourceScreen.js [hunk 1]
```diff
@@ -25,17 +26,9 @@ const ResourceScreen = () => {
 
 const styles = StyleSheet.create({
   container: {
```
Comment (mxnyee) [inline screens/ResourceScreen.js]: You can rename the styles object. Be sure to change the name inside the <Button> components too
```suggestion
  button: {
```

File: screens/ResourceScreen.js [hunk 2]
```diff
@@ -11,12 +11,13 @@ const ResourceScreen = () => {
...
     <Container>
     <Header leftButton="menu" onLeftButtonPress={() => Actions.drawerOpen()}>Resources</Header>
-
     <Content>
-      <View style={styles.container}>
-        <Text style={styles.welcome}>
-          Resource Screen
-        </Text>
+      <View>
+        <Button variant = "secondary" style = {styles.container}>naloxone injection instructions</Button>
```
Comment (mxnyee) [inline screens/ResourceScreen.js]: Similarly for the other buttons. See below
```suggestion
        <Button variant="secondary" style={styles.button}>naloxone injection instructions</Button>
```

File: screens/ResourceScreen.js [hunk 3]
```diff
@@ -11,12 +11,13 @@ const ResourceScreen = () => {
...
+        <Button variant = "secondary" style = {styles.container}>naloxone nasal spray instructions</Button>
+        <Button variant = "secondary" style = {styles.container}>call 911</Button>
+        <Button variant = "secondary"style = {styles.container}>your rights</Button>
+        <Button variant = "primary" style = {styles.container}>exit</Button>
```
Comment (mxnyee) [inline screens/ResourceScreen.js]: Redirect to the main screen like this
```suggestion
        <Button variant = "primary" style = {styles.container} onPress={() => Actions.main()}>exit</Button>
```

**Comment by Asiapenguin** [comment]
@matthewleemle please take a look at the comments when you got time

**Comment by matthewleemle** [comment]
> @matthewleemle please take a look at the comments when you got time

committed Maxine's

**Pull request merged**